bin/varnishncsa/varnishncsa.c

```diff
@@ -966,7 +965,8 @@ dispatch_f(struct VSL_data *vsl, struct VSL_transaction * const pt[],
 		skip = 0;
 		while (skip == 0 && 1 == VSL_Next(t->c)) {
 			tag = VSL_TAG(t->c->rec.ptr);
-			if (VSL_tagflags[tag])
+			if (VSL_tagflags[tag] &&
```

Comment:
don't we need to check the flags?
I would think this lead to filtered tags being printed also.

Reply:
have not checked though

Reply:
"filtered" as in "masked"? If so, we are good as the masked records are just not pushed into VSL.

In terms of tags, we have `UNSAFE` and `BINARY` which should be escaped, and `UNUSED` which is only useful in `vsl2rst.c`, so I didn't feel the need to be too granular, but I can change it if you prefer